### ISSUE TITLE
Error on windows: Error applying patch command rm -f doc/pdf/BUILD

### ISSUE THREAD

**Issue by schneiderlo** [nelhage/rules_boost#206]
Hello,

I am trying to use the boost library with Bazel on windows. Following the readme, my WORKSPACE file is as follow:

```
load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")

git_repository(
    name = "com_github_nelhage_rules_boost",
    commit = "1e3a69bf2d5cd10c34b74f066054cd335d033d71",
    remote = "https://github.com/nelhage/rules_boost",
    shallow_since = "1591047380 -0700",
)

load("@com_github_nelhage_rules_boost//:boost/boost.bzl", "boost_deps")
boost_deps()
```

and my src/BUILD file is:

```
load("//bazel:cc_wrapper.bzl", "cc_binary_wrapper")

cc_binary_wrapper(
  name = "main",
  srcs = ["main.cc"],
  deps = [
    "@boost//:config",
  ]
)
```

When running the command `bazel build //src:main` I obtain an error and the following stack:

```
INFO: Repository boost instantiated at:
  C:/users/schne/workspace/cpp-projects/myproject/WORKSPACE:15:11: in <toplevel>
  C:/users/schne/_bazel_schne/ybhurzp3/external/bazel_tools/tools/build_defs/repo/http.bzl:336:31: in <toplevel>
INFO: Repository 'boost' used the following cache hits instead of downloading the corresponding file.
 * Hash 'd73a8da01e8bf8c7eda40b4c84915071a8c8a0df4a6734537ddde4a8580524ee' for https://mirror.bazel.build/dl.bintray.com/boostorg/release/1.71.0/source/boost_1_71_0.tar.bz2
If the definition of 'boost' was updated, verify that the hashes were also updated.
ERROR: An error occurred during the fetch of repository 'boost':
   Traceback (most recent call last):
        File "C:/users/schne/_bazel_schne/ybhurzp3/external/bazel_tools/tools/build_defs/repo/http.bzl", line 121, column 10, in _http_archive_impl
                patch(ctx)
        File "C:/users/schne/_bazel_schne/ybhurzp3/external/bazel_tools/tools/build_defs/repo/utils.bzl", line 161, column 21, in patch    
                fail("Error applying patch command %s:\n%s%s" %
Error in fail: Error applying patch command rm -f doc/pdf/BUILD:
java.io.IOException: ERROR: src/main/native/windows/process.cc(202): CreateProcessW("bash" -c "rm -f doc/pdf/BUILD"): The specified file is not found
 (error: 2)
ERROR: C:/users/schne/workspace/cpp-projects/myproject/myproject/BUILD:3:18: //src:main depends on @boost//:config in repository @boost which failed to fetch. no such package '@boost//': Error applying patch command rm -f doc/pdf/BUILD:
java.io.IOException: ERROR: src/main/native/windows/process.cc(202): CreateProcessW("bash" -c "rm -f doc/pdf/BUILD"): The specified file is not found
 (error: 2)
ERROR: Analysis of target '//src:main' failed; build aborted: Analysis failed
INFO: Elapsed time: 102.100s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded, 0 targets configured)
```

How can I solve this problem ?

**Comment by Bioshock667** [comment]
Greeetings, I also faced the same issue as you did.  Fortunately, I was able to provide a quick fix that works for me here #210.  Please check out this PR and see if my branch fixes your problem.

**Comment by nelhage** [comment]
Should be closed by #210!

**Comment by schneiderlo** [comment]
It works, thank you !
